docs/getting-started-guides/ubuntu/bare-metal-romana.md

```diff
@@ -0,0 +1,65 @@
+---
+---
+
```

Comment:
3: * TOC
4: {:toc}

That way, your topic will have an in-page table of contents which I think is useful. :)